js/src/Ice/OutgoingConnectionFactory.js

```diff
@@ -148,32 +135,21 @@ export class OutgoingConnectionFactory {
         return promise;
     }
 
-    applyOverrides(endpts) {
-        const defaultsAndOverrides = this._instance.defaultsAndOverrides();
-        return endpts.map((endpoint) => {
-            if (defaultsAndOverrides.overrideTimeout) {
-                return endpoint.changeTimeout(defaultsAndOverrides.overrideTimeoutValue);
-            } else {
-                return endpoint;
-            }
-        });
-    }
-
     findConnectionByEndpoint(endpoints) {
         if (this._destroyed) {
             throw new CommunicatorDestroyedException();
         }
 
         Debug.assert(endpoints.length > 0);
 
-        for (let i = 0; i < endpoints.length; ++i) {
-            const endpoint = endpoints[i];
+        for (const endpoint of endpoints) {
```

Comment:
I would use the same names as in Java/C# etc.: proxyEndpoint for "for" variable, and endpoint for the fixed-up endpoint:
https://github.com/zeroc-ice/ice/blob/61662730fbe8845c8c9641a828585c4caf484a88/java/src/Ice/src/main/java/com/zeroc/IceInternal/OutgoingConnectionFactory.java#L282